### ISSUE TITLE
sap_hana_install: hdblcm configfile parameter 'verify_signature' is set incorrectly

### ISSUE THREAD

**Issue by berndfinger** [sap-linuxlab/community.sap_install#899]
The string after `sap_hana_install_` in the role variable `sap_hana_install_verify_signature` (`verify_signature`) is an hdblcm configfile parameter and therefore processed by the role for creating the hdblcm configfile. This causes this configfile parameter to be set to `True` or `False` instead of `y` or `n`.

The role does what we expect anyway (tested on SPS05, rev. 59.15 and SPS80, rev. 80):

- If the role variable `sap_hana_install_verify_signature` is set to the default of `false`, the hdblcm command line parameter `--verify_signature` is **not added** by the role, and the configfile parameter `verify_signature`, which is set to `False`, is interpreted by hdblcm as 'No'.

- If the role variable `sap_hana_install_verify_signature` is set to the alternative setting of `true`, the hdblcm command line parameter `--verify_signature` is **added** by the role, and the configfile parameter `verify_signature`, which is set to `True`, is ignored by hdblcm.

Such a behavior can be avoided by using another prefix (e.g. `sap_hana_install_cfg_`) for those role variables which are used for converting the hdblcm configfile template into a Jinja2 template, in task:
https://github.com/sap-linuxlab/community.sap_install/blob/c1b479ef710f0f0803a5a081eafd621465c86d4b/roles/sap_hana_install/tasks/pre_install/hdblcm_configfile.yml#L58

With this mechanism, the "role native" variables can be separated from those role variables which are used for directly setting the hdblcm configfile parameters.

**Comment by marcelmamula** [comment]
Yes, current implementation is too blind and it should be solved by either of these:
1. New dictionary like `sap_hana_install_configfile_custom`
```yaml
sap_hana_install_configfile_custom:
  verify_signature: y
```
2. New list of dictionary keys like `sap_hana_install_configfile_custom`
```yaml
sap_hana_install_configfile_custom:
  - key: verify_signature
    value: y
```

3. New variable prefix as you proposed `sap_hana_install_configfile_*`.

Either of these changes will work, but they are breaking changes and add backwards compatibility will be difficult.
